app/views/layouts/homepage.html.erb

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
```

Comment:
What's the difference between this and the layout upstream in sufia? It'd be useful to call that out with a comment when we need to update it in the future.